### PR TITLE
Added filters from filterbar to filters on offer page

### PR DESCRIPTION
**Pull request by MickhaelSh**
Added filters from filterbar to filters on offer page, created service to get price range for filter
also small fixes
image
image

### CODE REVIEW

File: src/containers/find-offer/filter-bar-menu/FilterBarMenu.tsx [hunk 1]
```diff
@@ -11,26 +11,32 @@ import ViewSwitcher from '~/components/view-switcher/ViewSwitcher'
...
-import { sortByFields } from '~/containers/find-offer/filter-bar-menu/FilterBarMenu.constants'
 
-import { BarMenuFilters, CardsViewTypes } from '~/types'
+import {
+  CardsView,
+  FindOffersFilters,
+  FindOffersUpdateFilter,
+  UserRoleEnum
+} from '~/types'
+import { sortTranslationKeys } from '../offer-filter-block/OfferFilterBlock.constants'
```
Comment (Tolik170) [inline src/containers/find-offer/filter-bar-menu/FilterBarMenu.tsx:21]: use alias

File: src/pages/find-offers/FindOffers.tsx [hunk 2]
```diff
@@ -10,55 +10,73 @@ import AppPagination from '~/components/app-pagination/AppPagination'
...
 import {
   mockOfferSquareCard,
   defaultFilters
 } from '~/pages/find-offers/FindOffers.constants'
+import { countActiveFilters } from '~/utils/count-active-filters'
+import { CardsViewEnum, CardsView, SizeEnum, UserRoleEnum } from '~/types'
+import OfferContainer from '~/containers/offerContainer/offerContainer'
 
 const FindOffers = () => {
+  const [currentPage, setCurrentPage] = useState(1)
```
[the remark this answers is not in the document]
Comment (MickhaelSh) [reply src/pages/find-offers/FindOffers.tsx:27]: this state will be deleted later, so i didn do anything with it

[the remark this answers is not in the document]
Comment (Tolik170) [reply src/pages/find-offers/FindOffers.tsx:27]: ok with this line of code, I also saw it in other places

File: src/containers/find-offer/filter-bar-menu/FilterBarMenu.tsx (hunk 1 above)
Comment (MickhaelSh) [reply src/containers/find-offer/filter-bar-menu/FilterBarMenu.tsx]: fixed

File: src/containers/find-offer/offer-filter-block/offer-filter-list/OfferFilterList.tsx [hunk 3]
```diff
@@ -9,21 +9,27 @@ import FilterInput from '~/components/filter-input/FilterInput'
...
+  ProficiencyLevelEnums
 } from '~/types'
+import useAxios from '~/hooks/use-axios'
+import { OfferService } from '~/services/offer-service'
 
 interface OfferFilterListProps {
   filters: FindOffersFilters
-  updateFilterByKey: (key: string) => (value: FindOfferFilterTypes) => void
-  updateFilter: FindOffersUpdateFilter
+  updateFilterByKey: <K extends keyof FindOffersFilters>(
```
[the remark this answers is not in the document]
Comment (MickhaelSh) [reply src/containers/find-offer/offer-filter-block/offer-filter-list/OfferFilterList.tsx]: fixed

File: src/pages/find-offers/FindOffers.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (MickhaelSh) [reply src/pages/find-offers/FindOffers.tsx]: fixed

**Review by Yurenko — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Review by mxrcury — APPROVED** [review]

**Pull request merged**